liferay-echo/liferay-portal
# LPS-186272 Add warning icon

https://issues.liferay.com/browse/LPS-186272

This PR depends on https://github.com/liferay-echo/liferay-portal/pull/12120. Just review from bcc034dc02c002cb720593b866687f57a74ef8b5 to 8c869638e45ea025090eef854fe18f532aa838fb.

**Testing steps**

1. Create a new site template: 'temp1'
2. Create a new site from the template: 'site1'
3. Add a new page to 'site1', name it "ConflictPage"
4. Add a new page to 'temp1', also named "ConflictPage"
5. Both of the friendly URLs will be "/conflictpage", therefore, they will be colliding.
6. Go to "Site Builder > Pages", verify that a "!" warning sign appears next to the page name. Tooltip on the warning shows more detailed info.
7. Go to "Home > Page Tree", verify that a "!" warning sign appears next to the page name. Tooltip on the page name shows more detailed info.
8. These warnings should work from the template's ([LPS-174465](https://issues.liferay.com/browse/LPS-174465)) and the site's ([LPS-174471](https://issues.liferay.com/browse/LPS-174471)) side as well, just with different tooltip messages.

**Feature flag**
`feature.flag.LPS-174471=true`
